File: src/main/kotlin/dev/jefy/connectpro/messaging/domain/repositoty/MessageRepository.java
package dev.jefy.connectpro.messaging.domain.repositoty;

import org.springframework.stereotype.Repository;

import dev.jefy.connectpro.messaging.domain.Message;
import dev.jefy.connectpro.messaging.domain.vo.MessageId;
import dev.jefy.connectpro.shared.infrastructure.ddd.AggregateRepository;

/**
 * @author Jôph Yamba
 */
@Repository
public interface MessageRepository extends AggregateRepository<Message, MessageId> {
}

File: src/main/kotlin/dev/jefy/connectpro/messaging/domain/repositoty/MessageRepository.kt
package dev.jefy.connectpro.messaging.domain.repositoty

import dev.jefy.connectpro.messaging.domain.Message
import dev.jefy.connectpro.messaging.domain.vo.MessageId
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * @author Jôph Yamba
 */
@Repository
interface MessageRepository : JpaRepository<Message, MessageId>
